tekton/ci/checks/check-pr-has-kind-label.yaml

```diff
@@ -82,7 +82,8 @@ spec:
             import yaml
             import sys
 
-            prLabels = json.loads('$(params.labels)')
+            prLabelsText = """$(params.labels)"""
+            prLabels = json.loads(prLabelsText)
             labelNames = list(map(lambda e: e["name"], prLabels))
```

Comment:
nit, why are you not using just list comprehension instead of functional programming heuristics, like map, lambdas and filters :)